### ISSUE TITLE
Incoherent required fields for maintenance windows without repetition

### ISSUE THREAD

**Issue by julienlafont-tabmo** [checkly/terraform-provider-checkly#151]
### Affected Resource(s)
Please list the resources as a list, for example:
- checkly_maintenance_windows

### Terraform Configuration Files

Here is the resource that need to be written when we want to create a **one-shot maintenance window (without repetition)**
```hcl
resource "checkly_maintenance_windows" "test" {
  name            = "Maintenance"

  starts_at       = "2022-05-05T18:00:00.000Z"
  ends_at         = "2022-05-06T06:00:00.000Z"
  repeat_ends_at  = ""
  repeat_interval = "0"
  repeat_unit     = "DAY"
}
```

### Expected Behavior
We should be able to create a maintenance without any "repeat" parameter at all. They are all optional from the UI.

### Actual Behavior
These 3 fields are required by terraform ([source](https://github.com/checkly/terraform-provider-checkly/blob/e8e271916052b23edb9026336c30fc0d7750bb42/checkly/resource_maintenance.go#L36-L47))
```
			"repeat_unit": {
				Type:     schema.TypeString,
				Required: true,
			},
			"repeat_ends_at": {
				Type:     schema.TypeString,
				Required: true,
			},
			"repeat_interval": {
				Type:     schema.TypeInt,
				Required: true,
			},
```

It's required to add weird default values to create a maintenance window without repetition.

**Comment by ianaya89** [comment]
Hi @julienlafont-tabmo, thanks for reporting this.
We are already aware of this bug and we are working on a fix

**Comment by ianaya89** [comment]
Resolved in latest release `v1.5.0-rc`
